src/components/Footer.jsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import styled from "styled-components";
+import colors from "../styles/colors";
+import Heart from "../styles/heart_icon.png";
+
+const FooterContainer = styled.div`
+  font-family: "Space Mono", monospace;
+  width: 100vw;
+  background-color: ${colors.black};
+`;
+
+const FooterContent = styled.div`
+  display: flex;
+  padding-top: 86px;
+  margin-left: 6vw;
+`;
+
+const FooterCol = styled.div`
+  margin-right: 8vw;
+  font-size: 20px;
```

Comment:
not sure if type sizes have been finalized yet but if they have, can you create type styles so you aren't assigning `font-size` every time? if not, just convert these to `ems` or `rems` for now